daemon/Libraries/Symlink/Symlink.cs

```diff
@@ -66,6 +66,25 @@ public bool IsSymlink(string linkPath)
             }
         }
 
-        public ISymlinkEnvironment Environment => _environment;
+        /// <summary>
+        /// Get the environment - interface friendly.
+        /// </summary>
+        /// <returns></returns>
+        public ISymlinkEnvironment GetEnvironment() => _environment;
+
+        /// <summary>
+        /// Get the process runner instance - interface friendly.
+        /// </summary>
+        /// <returns></returns>
+        public IProcessRunner GetProcessRunner() => _processRunner;
+
+        /// <summary>
+        /// Set the process runner for the class.
+        /// </summary>
+        /// <param name="processRunner"></param>
+        public void SetProcessRunner(IProcessRunner processRunner)
```

Comment:
Why a dedicated setter? Is the IProcessRunner expected to change?

Reply:
I needed to make the class interface friendly.